spec/takeaway_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'takeaway'
+require 'pry'
+
+describe Takeaway do
+
+  subject(:takeaway) { described_class.new(menu, order) }
```

Comment:
Text double hasn't been passed in here - so the double won't be used in the tests.